docs/roles/admin-tenant.json

```diff
@@ -0,0 +1,250 @@
+      {
+        "resource": "PERMISSION_GSLB",
+        "type": "NO_ACCESS"
+      },
+      {
+        "resource": "PERMISSION_CLOUD",
+        "type": "READ_ACCESS"
+      },
+      {
+        "resource": "PERMISSION_SERVICEENGINE",
+        "type": "WRITE_ACCESS"
+      },
+      {
+        "resource": "PERMISSION_SERVICEENGINEGROUP",
+        "type": "WRITE_ACCESS"
+      },
+      {
+        "resource": "PERMISSION_NETWORK",
+        "type": "WRITE_ACCESS"
+      },
+      {
+        "resource": "PERMISSION_GSLBGEODBPROFILE",
+        "type": "WRITE_ACCESS"
+      },
+      {
+        "resource": "PERMISSION_VRFCONTEXT",
+        "type": "WRITE_ACCESS"
+      },
+      {
+        "resource": "PERMISSION_APPLICATIONPROFILE",
+        "type": "WRITE_ACCESS"
+      },
+      {
+        "resource": "PERMISSION_NETWORKPROFILE",
+        "type": "WRITE_ACCESS"
+      },
+      {
+        "resource": "PERMISSION_APPLICATIONPERSISTENCEPROFILE",
+        "type": "WRITE_ACCESS"
+      },
+      {
+        "resource": "PERMISSION_HEALTHMONITOR",
+        "type": "WRITE_ACCESS"
+      },
+      {
+        "resource": "PERMISSION_ANALYTICSPROFILE",
+        "type": "WRITE_ACCESS"
+      },
+      {
+        "resource": "PERMISSION_IPAMDNSPROVIDERPROFILE",
+        "type": "WRITE_ACCESS"
+      },
+      {
+        "resource": "PERMISSION_CUSTOMIPAMDNSPROFILE",
+        "type": "WRITE_ACCESS"
+      },
+      {
+        "resource": "PERMISSION_TRAFFICCLONEPROFILE",
+        "type": "WRITE_ACCESS"
+      },
+      {
+        "resource": "PERMISSION_NATPOLICY",
+        "type": "NO_ACCESS"
+      },
+      {
+        "resource": "PERMISSION_L4POLICYSET",
+        "type": "WRITE_ACCESS"
+      },
+      {
+        "resource": "PERMISSION_HTTPPOLICYSET",
+        "type": "WRITE_ACCESS"
+      },
+      {
+        "resource": "PERMISSION_NETWORKSECURITYPOLICY",
+        "type": "WRITE_ACCESS"
+      },
+      {
+        "resource": "PERMISSION_AUTOSCALE",
+        "type": "WRITE_ACCESS"
+      },
+      {
+        "resource": "PERMISSION_DNSPOLICY",
+        "type": "WRITE_ACCESS"
+      },
+      {
+        "resource": "PERMISSION_IPADDRGROUP",
+        "type": "READ_ACCESS"
+      },
+      {
+        "resource": "PERMISSION_STRINGGROUP",
+        "type": "READ_ACCESS"
+      },
+      {
+        "resource": "PERMISSION_SSLKEYANDCERTIFICATE",
+        "type": "WRITE_ACCESS"
+      },
+      {
+        "resource": "PERMISSION_SSLPROFILE",
+        "type": "READ_ACCESS"
+      },
+      {
+        "resource": "PERMISSION_PKIPROFILE",
+        "type": "WRITE_ACCESS"
+      },
+      {
+        "resource": "PERMISSION_AUTHPROFILE",
+        "type": "READ_ACCESS"
+      },
+      {
+        "resource": "PERMISSION_PINGACCESSAGENT",
+        "type": "READ_ACCESS"
+      },
+      {
+        "resource": "PERMISSION_CERTIFICATEMANAGEMENTPROFILE",
+        "type": "READ_ACCESS"
+      },
+      {
+        "resource": "PERMISSION_HARDWARESECURITYMODULEGROUP",
```

Comment:
Is `READ` access required here? Does AMKO-admin do any operations related to it?